### PR TITLE
CFSS-HW1-MengdiHuang

### PR DESCRIPTION
**Pull request by mengdihuang**
It is definitely challenging for me to figure out the pull/push/pull request, however, I felt really rewarding when my results finally show up. I had no idea what a homework was like until I went to the open lab session and I am so glad that I did. The TAs were extremely helpful and they answered all my stupid questions. I am planning to use R for data analysis and graphs on my master thesis so I will definitely try really hard in this class.

### CODE REVIEW

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* It is a good practice to commit regularly like you did!
* You complete all the requirements and really did a great job :)
* Love that meme 💯

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |     X      |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |          |       X       |            |

Remarks:

* It is a good practice to commit regularly like you did!
* You complete all the requirements and really did a great job :)
* Love that meme 💯 
* Make sure you commit files through RStudio, not uploading through github. This will result in the reduction of point in the ease-of-access score in assignments.

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+
+
+## Name: Mengdi Huang
+* *University of Chicago*
+* **MAPSS**
+* My research interests:
```
Comment (jsimsspeyer) [inline README.md:6]: The bullets, italics, and bold text display nicely. Good work!

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+*My favorite animal is cat (Though I am a dog person)
+
+
+
+
+
+
+![Here's a picture of me trying to program in R](download-1.jpg)
+
+[This is the link to my old lab at UCSD and I am the one holding the cat!](http://oveislab.com/people)
```
Comment (jsimsspeyer) [inline README.md:28]: The link to the image works correctly

**Review by jsimsspeyer — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      *     |              |            |
| **Coding strategy**         |    *       |              |            |
| **Presentation**            |     *      |              |            |
| **Achievement, creativity** |     *      |              |            |
| **Ease of access**          |     *      |              |            |

Remarks:

* Good work implementing a variety of formatting
* Gave a good description of the difficulties you encountered while uploading
* Nice meme :)

**Comment by wanitchayap** [comment]
Just another note, make sure you commit files through RStudio, not uploading through github. This will result in the reduction of point in the ease-of-access score in subsequent assignments.

File: README.md [hunk 3]
```diff
@@ -1 +1,28 @@
...
+
+
+*My favorite animal is cat (Though I am a dog person)
+
+
+
+
+
+
+![Here's a picture of me trying to program in R](download-1.jpg)
```
Comment (tgriffin05) [inline README.md:26]: Mengdi, 

Your image was embedded nicely in RMarkdown. Your image was creative and funny! I also love cats!

File: README.md [hunk 4]
```diff
@@ -1 +1,28 @@
...
+* *University of Chicago*
+* **MAPSS**
+* My research interests:
+1. Social Cognition
+1. Emotions
+1. Behavioral Science
+
+* I got interested in psychology when I was in high school, when I was a volunteer teaching assistant in an autism school. I interacted with a little patient called Dongdong. Though he was very reluctant to interact with me at first, I became his favorite teacher in the end. This experience made me wonder that why patients with autism are often emotionally unstable and what I can do for them. Therefore I decided to study psychology during my undergraduate life.
+
+* Aside from psychology, here is a little fun facts about me:
```
Comment (tgriffin05) [inline README.md:13]: I like how you incorporated both bullet and number list in your file. The variety made your document easy to read!

**Comment by tgriffin05** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x    |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |          x |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |         x  |              |            |

Remarks:

* The meme was creative and engaging!
* Your list were numbered and bulleted clearly
* Your frequent commits made it easy to review the change of your homework overtime

File: README.md (hunk 2 above)
Comment (wanitchayap) [inline README.md:28]: Love this!

**Pull request closed without merging**